map/local_data.html

```diff
@@ -65,6 +85,24 @@ <h1>BCU Labs - Data Team</h1>
             type: 'geojson',
             data: 'plots/LTS.json'
         });
+        const colors = [
+            'black',
```

Comment:
@sckilcoyne what are black and white used for here? I wasn't able to figure that out when looking through the code

Reply:
I'm pretty sure this is an artifact from the code I forked, but I think it is to plot ways that have an LTS that isn't 1-4. I have kinda assumed when I have looked at it that ideally nothing plots with those colors but if something could be plotted with those colors it would be better for debugging to make it apparent than to ignore that. So I've left it in and not messed with anything as a form of regression test. Not 100% that actually is a good idea though.